File: 10 - Python/Exercícios/02 - Estrutura Condicional/Exercício 11.py
#Exercício - Aumento

salarioAtual = float(input("Digite o salário atual: R$"))

if salarioAtual <= 1000:
    porcentagem = 20
elif salarioAtual <= 3000:
    porcentagem = 15
elif salarioAtual <= 8000:
    porcentagem = 10
else:
    porcentagem = 5

aumento = salarioAtual * (porcentagem / 100)

salarioFinal = salarioAtual + aumento

print("Novo salário = R$", round(salarioFinal,2))
print("Aumento = R$", round(aumento,2))
print("Porcentagem = ", porcentagem, "%")

File: 06 - C/Exercícios/02 - Estrutura Condicional/Exercício 11.c
//Exercício - Aumento

#include <stdio.h>

int main() {

    double salarioAtual, salarioFinal, aumento, porcentagem;

    printf("Digite o salario da pessoa: ");
    scanf("%lf", &salarioAtual);

    if (salarioAtual <= 1000){
        porcentagem = 20.0;
    }
    else if (salarioAtual <= 3000){
        porcentagem = 15.0;
    }
    else if (salarioAtual <= 8000){
        porcentagem = 10.0;
    }
    else{
        porcentagem = 5.0;
    }

    aumento = salarioAtual * (porcentagem / 100);
    salarioFinal = salarioAtual + aumento;

    printf("Novo salario = R$%.2lf\n", salarioFinal);
    printf("Aumento = R$%.2lf\n", aumento);
    printf("Porcentagem = %.lf %%", porcentagem);

    return 0;

}